client/reader/tags/alphabetic-tags.tsx

```diff
@@ -87,6 +89,23 @@ const scrollToLetter = ( letter: string ) => {
 
 export default function AlphabeticTags( { alphabeticTags }: AlphabeticTagsProps ) {
 	const translate = useTranslate();
+	const [ isSticky, setIsSticky ] = useState( false );
+
+	useEffect( () => {
+		const handleScroll = () => {
+			const tableA = document.getElementById( 'alphabetic-tags-table-A' );
```

Comment:
I think this needs to be done with a ref so that react is guaranteed to recognize and load it properly. I'll take a stab at changing it.